CSharpMath/Display/AttributedGlyphRun.cs

```diff
@@ -7,9 +8,12 @@ namespace CSharpMath.Display {
   /// over the whole string. We use KernedGlyph objects instead of Glyphs to
   /// allow us to set kern on a per-glyph basis.</summary>
   public class AttributedGlyphRun<TFont, TGlyph> where TFont : FrontEnd.IFont<TGlyph> {
-    public AttributedGlyphRun(string text, IEnumerable<TGlyph> glyphs, TFont font, bool isPlaceHolder = false) {
+    public AttributedGlyphRun(string text, IEnumerable<TGlyph> glyphs, TFont font, bool isPlaceHolder = false, Color? foreColor = null) {
```

Comment:
`isPlaceHolder` and `foreColor` can probably be replaced with one `placeholderColor`

Reply:
The placeholder has a default Color null. If the argument placeholderColor is null, there is no way to distinguish between "placeholder with default color" vs "is no placeholder'.